compiler/src/test/scala/play/twirl/compiler/test/CompilerSpec.scala

```diff
@@ -87,12 +87,36 @@ abstract class CompilerTests(oldParser: Boolean = false) extends Specification {
       set must be_==("firstsecondthird")
     }
 
-    "compile successfully (imports)" in {
```

Comment:
@jroper: here's the test you were asking for. It was already there, I just hadn't run it. You're right that my changes had broken this use case. The new code should handle both this use case and also resolve the issue with ambiguous wildcard imports.